db/migrate/20220127164323_add_feedback_email_to_local_authorities.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class AddFeedbackEmailToLocalAuthorities < ActiveRecord::Migration[6.1]
+  def up
+    add_column :local_authorities, :feedback_email, :string
+
+    LocalAuthority.find_each do |authority|
+      authority.subdomain == "lambeth" && authority.update(feedback_email: "digitalplanning@lambeth.gov.uk")
+      authority.subdomain == "southwark" && authority.update(feedback_email: "digital.projects@southwark.gov.uk")
+      if authority.subdomain == "buckinghamshire"
+        authority.update(feedback_email: "planning.digital@buckinghamshire.gov.uk")
+      end
```

Comment:
is this an `if` block unlike the first two updates just because of rubocop?

Reply:
Yes 😂

Reply:
haha well it does the same thing anyway so should be fine